.github/ISSUE_TEMPLATE/blocker.yaml

```diff
@@ -46,12 +46,10 @@ body:
       options:
         - label: cbuild
           required: false
-        - label: cbuild-gen
+        - label: csolution
           required: false
         - label: cpackget
           required: false
-        - label: csolution
-          required: false
```

Comment:
I would recommend to replace `cbuild-gen` by `cbuild2cmake` and keep `csolution` where it is, considering the tools are in alphabetical order.